### PR TITLE
Change Response for Solution PUT call

### PR DESCRIPTION
**Pull request by dashimi16**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json [hunk 1]
```diff
@@ -69,8 +69,8 @@
                     }
                 ],
                 "responses": {
-                    "200": {
-                        "description": "OK response definition.",
+                    "201": {
```
Comment (dsgouda) [inline specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Please verify if this is a long running operation

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json (hunk 1 above)
Comment (dashimi16) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Yes, this is a long running operation.

Comment (dsgouda) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: Please mark it as long running (`"x-ms-long-running-operation":true`)
While we are at it please confirm whether the `DELETE` operation is long running too and update it

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json [hunk 2]
```diff
@@ -40,8 +40,9 @@
                     "Solution"
                 ],
                 "x-ms-examples": {
-                     "SolutionCreate": { "$ref": "./examples/SolutionCreate.json" }
+                    "SolutionCreate": { "$ref": "./examples/SolutionCreate.json" }
                 },
+                "x-ms-long-running-operation": "true",
```
Comment (dsgouda) [inline specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json]: `true` is a boolean value, please set this as 
```json
"x-ms-long-running-operation":  true
```

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Review by dsgouda — APPROVED** [review]

**Comment by dsgouda** [comment]
LGTM

**Pull request merged**